error/README.md

```diff
@@ -88,16 +88,16 @@
 | Image Verifier | Verifier | FmcLoadAddrUnaligned |0x000B0022 |
 | Image Verifier | Verifier | FmcEntryPointInvalid |0x000B0023 |
 | Image Verifier | Verifier | FmcEntryPointUnaligned |0x000B0024 |
-| Image Verifier | Verifier | FmcSvnGreaterThanMaxSupported |0x000B0025 |
-| Image Verifier | Verifier | FmcSvnLessThanMinSupported |0x000B0026 |
-| Image Verifier | Verifier | FmcSvnLessThanFuse |0x000B0027 |
+| Image Verifier | Verifier | (Deprecated) FmcSvnGreaterThanMaxSupported |0x000B0025 |
```

Comment:
Can rebase to pick up https://github.com/chipsalliance/caliptra-sw/pull/1779 and drop changes in this file.

Reply:
That's in main, not main-2.x. Will drop changes in this file anyways.